### ISSUE TITLE
connections to/from local networks not shown

### ISSUE THREAD

**Issue by rade** [weaveworks/scope#1846]
Scope fails to show connections where one endpoint is on a network that is deemed "local", i.e. has an IP that is in a subnet of one of the network interfaces' configured subnets.

To reproduce, I fired up two EC2 VMs, ran scope in one, connecting to Weave Cloud, plus a 'nc' listener. And have the security group configured to allow access to the port from the outside. I then connected from another VM. The connection is working just fine, but nothing shows up in scope.

By contrast, connecting locally makes both processes show up in the topology, with an edge between them. And when connecting from the internet, I get an edge from the 'The Internet (Inbound connections)' to the 'nc' node.

**Comment by rade** [comment]
It's actually not clear how we should represent these connections. Lump them in with The Internet? Have separate "LAN (Inbound Connections)" and "LAN (Outbound Connections)" nodes?

**Comment by rade** [comment]
Hmm. I wonder whether this issue has arisen as a result of #1326.

**Comment by tomwilkie** [comment]
Yes, lan connections == non internet pseudo nodes.

I disabled this as we can reliably recognise these edges from edges that
should be going to a process/machine/container we know about.  Blame
procspy.

**Comment by rade** [comment]
> I disabled this as we can reliably recognise these edges from edges that should be going to a process/machine/container we know about.

Right, but that ends up completely omitting connections where one endpoint is on a "LAN" host which isn't running scope. e.g. connections from an ELB.

**Comment by 2opremio** [comment]
> Right, but that ends up completely omitting connections where one endpoint is on a "LAN" host which isn't running scope. e.g. connections from an ELB.

I actually started experiencing this  in the ECS demo a few versions ago but I thought it was by design ...
